null_safety_examples/misc/lib/language_tour/callable_classes.dart

```diff
@@ -7,4 +7,4 @@ class WannabeFunction {
 var wf = WannabeFunction();
 var out = wf('Hi', 'there,', 'gang');
 
-main() => print(out);
+void main() => print(out);
```

Comment:
If we do this, can we remove the ignore_for_file comment?

Reply:
This is also used for the `wf` and `out` as they are considered public APIs. I can remove the ignore and type annotate those if desired though.

Reply:
Nope, that's fine. Never mind. :)